lms/djangoapps/oauth2_handler/tests.py

```diff
@@ -158,27 +159,46 @@ def get_with_claim_value(self, scope, claim, values):
         self.assertEqual(result.status_code, 200)
         return claims
 
-    def test_request_staff_courses_using_scope(self):
-        CourseStaffRole(self.course_key).add_users(self.user)
+    def test_request_global_staff_courses_using_scope(self):
+        GlobalStaff().add_users(self.user)
         claims = self.get_with_scope('course_staff')
```

Comment:
Decorate with `check_mongo_calls` here so (1) it's clear that this functionality calls mongo, while the others don't and (2) to verify the check_mongo_calls returns correctly.